### DISCUSSION TITLE
Datetime format, MS SQL Table

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#9451] [Help] [answered]
Hi,

I would like to save the date & time with `{{ date }}` but the timestamp is always 00:00.
How can I save the time as well?

Sincerely,
K

**Comment by melohagan** [comment]
Hey @kasgithub8 

You can provide a format as a string, e.g. `{{ date "yyyy-MM-dd HH:mm:ss" }}`

**Reply by kasgithub8** [reply]
> You can provide a format as a string, e.g. `{{ date "yyyy-MM-dd HH:mm:ss" }}`

@melohagan 
![image](https://user-images.githubusercontent.com/95284278/214864813-5ad8af92-2460-4b41-bf5a-124679173db7.png)
It doesn't get flagged with an error when you use the provided helper `{{ date now "..." }}` but it doesn't display anything.

**Reply by melohagan** [reply]
Ah yes, it would be `date now ...`

Are you simply trying to display that in a paragraph component?

**Reply by kasgithub8** [reply, marked as answer]
@melohagan ah, I got it.
The Problem was that when you put `{{ date }}` as the default value (date picker), it shows the date.
When you put `{{ date now "YYYY-MM-d HH:mm:ss" }}` as the default value, it just shows the placeholder of that component but the default value is the actual date & time.

**Reply by AlejandroFNadal** [reply]
Just in case someone was trying to use this to modify a table column:

{{ date Value "YYYY-MM-DD"}} 

Value is literally the word Value. Do not put: {{ date {{Value}} "YYYY-MM-DD"}}. This will not work.

**Reply by melohagan** [reply]
Thanks @AlejandroFNadal in Handlebars syntax you will only ever have two sets of curly braces, i.e. `{{ <expression> }}`.

If you want to include a subexpression then you'll want to use parenthesis: https://handlebarsjs.com/guide/expressions.html#subexpressions
